### DISCUSSION TITLE
Why aren't disclosure carets appearing?

### DISCUSSION THREAD

**Discussion by oliverlovelltas** [vslinko/obsidian-outliner#193] [Q&A] [answered]
Hi. 

Disclosure carets are not appearing after installing obsidian outliner. 

**To reproduce**
1. Download Outliner
2. Create a bullet point list
3. See that disclosure carets are not displaying

**Expected behaviour**
After installing outliner it becomes possible to collapse and expand lists with disclosure carets in both edit and preview mode. 

**Screenshots**
Outliner is installed
![image](https://user-images.githubusercontent.com/29011009/130389065-d59c52c7-4b3a-44d2-8f59-36b47df64a88.png)

List is not collapsible
![image](https://user-images.githubusercontent.com/29011009/130389101-19fb39af-4816-4134-9f76-86b5924d0d8f.png)

**Desktop**
 - OS: Mac OS 11.4, and Windows 10
 - Obsidian Version: 0.12.12
 - Outliner 1.3.1

**Additional info.**
I have tried restarting computer, tried on multiple devices (Mac OS 11.4 and Windows 10), and quit and restarted obsidian. A friend has also just reported this same issue to me.

**Comment by alemsabic** [comment]
Same here. Stopped working after changing vault. Mac OS 11.4. Removed all plugins, changed folder, restarted obsidian -- nothing helps. :-)

**Comment by alemsabic** [comment]
@oliverlovelltas After reinstalling a lot of plugins, I forgot to check this little box. After that, everything - magically! - reappeared.
:-D

![Screenshot at Aug 26 16-55-46](https://user-images.githubusercontent.com/9151275/130987241-40c0275d-f80f-4f17-851c-c1528b71e83a.png)

**Comment by oliverlovelltas** [comment]
Thanks for sharing!

Unfortunately, that doesn't add the carets for me, it only adds the styling. 

Does it add the carets for you?

![image](https://user-images.githubusercontent.com/29011009/131024117-938eebb7-7c41-4614-b8e6-b1787da656de.png)

I did work out a weird workaround though. I duplicated one of my old vaults with Outliner 1.2.1 in it (where the disclosure carets were working), and used that as the new vault, and that works!

(It's also weird that this works, but that installing Outliner 1.2.1 directly doesn't work)

If you'd like to try out this workaround, here's the duplicated vault that worked for me :)

https://www.dropbox.com/sh/un34hxaxflb78eu/AAC9YlK0ka7EP-Sc-qZ6kLZNa?dl=0

I sent this folder to a friend via dropbox and the workaround worked for him too. So strange.

Cheers,

O.

**Comment by alemsabic** [comment, marked as answer]
@oliverlovelltas Check this one too - and it will work. (You want your headings to be collapsible too, it's the point above this one. Have a nice day, my friend. (And thanks for the dropbox link.) 
![Screenshot at Aug 27 08-02-01](https://user-images.githubusercontent.com/9151275/131079780-ca9fae93-66e1-438e-8965-84a8ece66b8e.png)

**Comment by oliverlovelltas** [comment]
Wow, great, thanks so much!
